shell.sh

```
@@ -0,0 +1,34 @@
+
+while true; do
+    echo "This loop will run forever"
+done
+
+echo "This line should have a shebang at the start"
+
+touch -c file.txt
+
+cp /path/to/source.txt /path/to/destination.txt
+
+read -p "Enter your name: " user_name
```

Shellcheck

[info] read without -r will mangle backslashes.

(SC2162)